addons/sale_project/views/project_task_views.xml

```diff
@@ -100,4 +100,32 @@
         </field>
     </record>
 
+    <record id="project_milestone_view_form" model="ir.ui.view">
+        <field name="name">project.milestone.view.form.inherit</field>
+        <field name="model">project.milestone</field>
+        <field name="inherit_id" ref="project.project_milestone_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='main_details']" position="after">
+                <group t-if="record.allow_billable.raw_value">
+                    <field name="project_partner_id" invisible="1"/>
+                    <field name="sale_line_id"/>
```

Comment:
For this field, I guess we need to do the same logic than the one in project and task. That is:
1) Not allow the create
2) not allow the user to open the form view of the SOL if he is not a salesman.

See [this commit](https://github.com/odoo/enterprise/commit/98a2f5ac4ab772e1545f898f16a449e3742609e1) for instance. 🙂

Reply:
Done